docs/Configuration/BitBucket.md

```diff
@@ -47,20 +47,88 @@ If you are behind a corporate firewall or VPN you may need to utilize a proxy se
+
+<!-- ![image](https://user-images.githubusercontent.com/3294100/220094172-9e8e9e8b-75ea-4c3e-8e5b-716320dabb64.png) -->
 
 
 #### Test and Save Connection
 
 Click `Test Connection`, if the connection is successful, click `Save Connection` to add the connection.
 
-
 ### Step 2 - Configure Blueprint
 
-Similar to other beta plugins, BitBucket does not support `project`, which means, you can only collect BitBucket data via blueprint's advanced mode. 
+![image](https://user-images.githubusercontent.com/3294100/220236338-772b30b7-974f-4bc2-89ce-f2abe5e92a5e.png)
+
+#### Repositories
+
+Select the BitBucket repos to collect.
+
+#### Data Entities
+
+Usually, you don't have to modify this part. However, if you don't want to collect certain BitBucket entities, you can unselect some entities to accelerate the collection speed.
+
+- Issue Tracking: BitBucket issues, issue comments, etc.
+- Source Code Management: BitBucket repos, refs, commits, etc.
+- Code Review: BitBucket PRs, PR comments, etc.
+- CI/CD: BitBucket Pipelines, BitBucket Deployments, etc.
+- Cross Domain: BitBucket accounts, etc.
 
 Please go to the `Blueprints` page and switch to advanced mode. See how to use advanced mode and JSON [examples](AdvancedMode.md).
 
+### Step 3 - Adding Transformation Rules (Optional)
+
+![image](https://user-images.githubusercontent.com/3294100/220338276-a67cd8cc-ea76-4cb2-bb7b-bba581d21d70.png)
+
+Without changing default transformation rules, you can still view the Metrics dashboard. However, if you want to view pre-built dashboards, the following transformation rules, especially "Type/Bug", should be added.<br/>
+
+Each BitBucket repo has at most ONE set of transformation rules.
+
+#### Issue Tracking
+
+- TODO: the issues with selected states can be recognized not start issues. 
+
+- IN-PROGRESS: The issues statuses that indicate an issue is work in progress.
+- DONE: The issue statuses that indicate an issue is completed.
+- OTHER: Other issues statuses that can not be mapped to the above three statuses.
+
+#### CI/CD
+
+This set of configurations is used for calculating [DORA metrics](../DORA.md).
+
+BitBucket has several key CI entities: `pipelines`, `pipeline steps`, and `deployments`. A Bitbucket pipeline contains several pipeline steps. Each pipeline step defined with a deployment key can be mapped to a BitBucket deployment.
+
+Each Bitbucket pipeline is converted to a cicd_pipeline in DevLake's domain layer schema and each Bitbucket pipeline step is converted to a cicd_task in DevLake's domain layer.
+![image](https://user-images.githubusercontent.com/3294100/220288225-71bee07d-c319-45bd-98e5-f4d01359840e.png)
+
+![image](https://user-images.githubusercontent.com/3294100/220289726-3909d368-1414-456c-a527-12a693745611.png)
```

Comment:
1. Annotation change: If a pipeline step defines `deployment` with a value (usually means envrionment), this pipeline step is also a BitBucket deployment. Their relationship is 1:1
![image](https://user-images.githubusercontent.com/14050754/221522469-85ee8ffe-c8ee-41cc-ad3d-3df6204319eb.png)

2. Add a screenshot of how to define deployment and environment in `.yaml` to explain point 1. For example,
![1g9BrGdVlb](https://user-images.githubusercontent.com/14050754/221524899-4de36a2e-f548-4844-84f3-625b0c83797e.jpg)

Reply:
I add 1 in doc and add a notice in pic.